man/man8/zfsprops.8

```diff
@@ -962,6 +962,103 @@ Unless necessary, deduplication should NOT be enabled on a system. See the
 section of
 .Xr zfsconcepts 8 .
 .It Xo
+.Sy directio Ns = Ns Sy off Ns | Ns Sy on Ns | Ns Sy strict Ns | Ns
+.Sy legacy
+.Xc
+Sets the policy for Direct IO when an IO request uses the
+.Sx O_DIRECT
+flag. The default value is
+.Sy strict.
+.Pp
+All Direct IO requests must be page-aligned.
+.Pp
+If this property is set to
+.Sy off
+all Direct IO requests will always fail.
```

Comment:
"fail" means that the read or write syscall with return ENOTSUP, right?

Reply:
That is correct.